docs/modules/globalfee.md

```diff
@@ -64,111 +119,104 @@ A `proposal.json` example:
-- fees="2000000 *0uatom, 2000000* 0.5stake", fail, (this is due to [fee parsing, zero coins are removed](https://github.com/cosmos/cosmos-sdk/blob/e716e4103e934344aa7be6dc9b5c453bdec5f225/client/tx/factory.go#L144), it equals to paidfees = 0.5stake in this case)
-- paidfees="", pass
-- paidfees="2000000 * 1uatom, 0.5stake", pass
-- paidfees="0uatom, 0stake" pass, (due to the parsing of paidfees, it makes paidfees="")
-- paidfees="2000000 * 1stake", pass
-
-- **Case 6**: globalfee=[0.001uatom, 1stake], min_gas_prices=0.002uatom, gas=200000.
-  - paidfee = "2000000 * 0.0002uatom", pass
-  - paidfee = "2000000 * 0.0001uatom", fail
-  - paidfee = "2000000 * 1stake", pass
-  - paidfee = "2000000 * 1/2stake", fail
-  - paidfee = "2000000 *0.0001uatom, 2000000*1stake", pass
-  - paidfee = "2000000 *0.0002atom, 2000000*1/2stake", pass
-  - paidfee = "2000000 *0.0001uatom, 2000000*1/2stake", fail
+### Case 6
+
+**Setting:** globalfee=[0.001uatom, 1stake], minimum-gas-prices=0.002uatom, gas=200000.
+
+Note that the required amount of `uatom` in globalfee is overwritten by the amount in minimum-gas-prices. 
+
+  - paidfee = "2000000 * 0.002uatom", `pass`
+  - paidfee = "2000000 * 0.001uatom", `fail` (insufficient funds)
+  - paidfee = "2000000 * 1stake", `pass`
+  - paidfee = "2000000 * 0.5stake", `fail` (insufficient funds)
+  - paidfee = "2000000 * 0.001uatom, 2000000 * 1stake", `pass`
+  - paidfee = "2000000 * 0.002atom, 2000000 * 0.5stake", `pass`
+  - paidfee = "2000000 * 0.001uatom, 2000000 * 0.5stake", `fail` (insufficient funds)
   
-- **Case 7**:globalfee=[0.0001uatom], min_gas_prices=0.0002uatom,1stake, gas=200000.
+### Case 7
+
+**Setting:** globalfee=[0.1uatom], minimum-gas-prices=0.2uatom,1stake, gas=200000, bypass-min-fee-msg-types = ["/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward"]
+
+Note that the required amount of `uatom` in globalfee is overwritten by the amount in minimum-gas-prices. 
+Also, the `1stake` in minimum-gas-prices is ignored.
+
+  - msg withdraw-all-rewards with paidfee="", `pass`
+  - msg withdraw-all-rewards with paidfee="200000 * 0.05uatom", `pass`
+  - msg withdraw-all-rewards with paidfee="200000 * 1stake", `fail` (unexpected denom)
+
+### Case 8
+
+**Setting:** globalfee=[1uatom], minimum-gas-prices="", gas=300000, bypass-min-fee-msg-types = ["/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward"]
 
-   `bypass-min-fee-msg-types = ["/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward"]`
-  - msg withdraw-all-rewards with paidfee=0uatom, pass
-  - msg withdraw-all-rewards with paidfee=200000 * 0.0001/2uatom, pass
-  - msg withdraw-all-rewards with paidfee=0stake,0photon, pass
-  - msg withdraw-all-rewards with paidfee=200000 * 1stake, 0photon, fail
+  - msg withdraw-all-rewards with paidfee="", `fail` (gas limit exceeded for bypass transactions)
+  - msg withdraw-all-rewards with paidfee="300000 * 0.5uatom", `fail` (gas limit exceeded for bypass transactions)
```

Comment:
I think this will pass, ` withdraw-all-rewards` is  200_000 ?

Reply:
It will not pass as the gas is 300.000 and the threshold is set to 200.000. Therefore, the fee must be at least 300000 * 1uatom. So maybe add the following:
```suggestion
  - msg withdraw-all-rewards with paidfee="300000 * 0.5uatom", `fail` (gas limit exceeded for bypass transactions, insufficient funds)
```